pkg/kubelet/cm/dra/manager.go

```diff
@@ -150,6 +151,10 @@ func (m *ManagerImpl) reconcileLoop(ctx context.Context) {
 // for each new resource requirement, process their responses and update the cached
 // containerResources on success.
 func (m *ManagerImpl) PrepareResources(ctx context.Context, pod *v1.Pod) error {
+	metricLabel := "PrepareResources"
+	defer func(startTime time.Time) {
+		metrics.DRAOperationsDuration.WithLabelValues(metricLabel).Observe(time.Since(startTime).Seconds())
```

Comment:
It will be great to add the dimension whether the operation failed or not. So we can see how long successful operations are and not contaminate data with timeouts (very long) or connection errors (super short).

Reply:
done